RELEASE/scripts/autoscend/combat/auto_combat_default_stage3.ash

```diff
@@ -127,6 +127,19 @@ string auto_combatDefaultStage3(int round, monster enemy, string text)
 			}
 		}
 
+		// use red rocket to get 5x stats from next food item consumed. Staggers on turn used.
+		if(canUse($item[red rocket]) && have_effect($effect[Everything Looks Red]) <= 0 && have_effect($effect[Ready to Eat]) <= 0)
+		{
+			//use if next food is large in size. Currently autoConsume doesn't analyze stat gain, which would be better
+			//assume next food will be eaten within 100 turns
```

Comment:
add a `my_adventures() < 100` to the if and it will not be an assumption anymore but a fact

Reply:
Good idea. Looks like consumeStuff fills liver first. Added a check for liver as well as my_adventures. Not perfect since consumeStuff only fills up to 10 inebriety if we aren't finished with billiards room, but is an improvement.